core/src/test/java/uk/gov/gchq/koryphe/impl/function/LengthTest.java

```diff
@@ -131,23 +131,23 @@ public void shouldReturnLengthForListInput() {
         final Integer result = function.apply(input);
 
         // Then
-        assertEquals(new Integer(4), result);
+        assertEquals(Integer.valueOf(4), result);
     }
 
     @Test
     public void shouldReturnLengthForSetInput() {
         // Given
         final Length function = new Length();
         final Set<Object> input = new HashSet<>();
-        input.add(2.718);
-        input.add(3.142);
+        input.add(2.142);
```

Comment:
Why have these number been switched around?

Reply:
Spotbugs notices that `3.142` is an approximation of π and throws an error to warn you that an inexact value of π is being used instead of Math.PI. Rather than adding an exception to the spot bugs-exclude.xml list, I opted to change these constants to make them appear like boring numbers (not π and e).